x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/aggregation/GroupingAggregatorFunctionTestCase.java

```diff
@@ -200,25 +200,49 @@ protected final void assertSimpleOutput(List<Page> input, List<Page> results) {
         assertSimpleOutput(input, results, true);
     }
 
-    private void assertSimpleOutput(List<Page> input, List<Page> results, boolean assertGroupCount) {
+    private void assertSimpleOutput(List<Page> input, List<Page> results, boolean expectExactGroups) {
```

Comment:
Previously a `false` here skipped a bunch of checks. Now it just weakens them - testing that we get the expected groups and MAYBE more. If this is `true` we still check for exactly the matching groups.